arangodb-foxx-services/src/main/index.ts

```diff
@@ -17,5 +17,7 @@
 import router from './routes'
 import { context } from '@arangodb/locals'
 
+import './modules'
```

Comment:
@wajda I do recommend decreasing Doorway Effect (the best practice is prefer less amount of subfolders)

Reply:
Less? `modules` is a actually a 1st level folder under the Foxx service source root, which is `/arangodb-foxx-services/src/main` in the project structure.

Reply:
What's you suggestion?
The idea here is this - we have an core services, and a set of optional loosely coupled modules that suppose to subscribe for system events and do their work not requiring the core part of the system to be aware of their existence. But somehow those modules have to be called for the first time to be able to register themselves. My solution to this is only import `modules/` folder in the core part of the app, and then the rest of the modules would be registered via imports in that umbrella `index.ts`. So when a new module is added to or deleted from the system, all the needs to be done is just to create/delete a module folder and add/remove the respective `import` statement from the `modules/index.ts`. This way the core system is kind of remains unaware of the modules, while technically still execute them indirectly. How would you do it without this indirection layer?

Reply:
webpack?